src/sgame/sg_bot_nav.cpp

```diff
@@ -797,10 +804,32 @@ Global Bot Navigation
 bool BotMoveToGoal( gentity_t *self )
 {
 	glm::vec3 dir;
-	if ( BotAvoidObstacles( self, dir ) )
+	constexpr int ignoreGeometryThreshold = 1700; // 1.7s, we seem stuck
```

Comment:
Note that stuck time is only recalculated once per second, so it seems a little fishy to use such a low threshold for something.

Reply:
Well, it seems to work well enough like that. I guess that would mean 1.7 to 2.7s then, but I don't think that's a problem.

Reply:
Maybe you could change stuck time to be recalculated every 500ms or something so that it would be more reliable.

LGTM